README.md

```diff
@@ -203,37 +216,113 @@ curl -v -X POST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid
     }'
 ```
 
-### Delete an Indoor Asset Set
+### Getting all the Indoor Assets from a floor
+
+You can also fetch all the assets for a given floor you have access to.  Using a dev token will show both Live and Staged sets you have access to:
 
 ```sh
-curl -v -X DELETE https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>?token=<dev_auth_token>
+curl -v https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entities.json?token=<dev_auth_token>
 ```
 
-### Publish an Indoor Asset Set
+Using an api-key instead of a dev token will only show you the Live assets you have access to:
 
 ```sh
-curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/publish?token=<dev_auth_token>
+curl -v https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entities.json?apikey=<apikey_with_access_to_indoor_map>
 ```
 
-For more information on publishing an Indoor Asset Set, see the [Indoor Assets Tutorial](TUTORIAL-ASSETS.md#publishing-your-assets).
+You can also change the extension to .hcff to retrieve a chunk file of the same data, or .geojson to download the assets + the floor trace json combined.
+
+### Asset Sets
```

Comment:
I think Asset Sets should come before Assets, as many of the asset APIs refer to asset sets.